requirements.txt

```diff
@@ -0,0 +1,7 @@
+requests == 2.10
+pexpect == 2.4
+urllib3 == 1.20
+pyOpenSSL == 0.14
+cryptography == 1.3.4
+idna == 2.0.0
+
```

Comment:
other projects will not be using the requirements.txt file for dxskytap. They will have their own requirements.txt file. In that situation the other project just needs to meet the setup.py.  This requirements.txt is only used and needed when testing dxskytap.